### ISSUE TITLE
select locale when creating user via cli

### ISSUE THREAD

**Issue by ClaudiuCreanga** [magento/magento2#6768]
`php bin/magento admin:user:create --help
`
gives:

```
 --admin-user           (Required) Admin user
 --admin-password       (Required) Admin password
 --admin-email          (Required) Admin email
 --admin-firstname      (Required) Admin first name
 --admin-lastname       (Required) Admin last name
 --magento-init-params  Add to any command to customize Magento initialization parameters
                        For example: "MAGE_MODE=developer&MAGE_DIRS[base][path]=/var/www/example.com&MAGE_DIRS[cache][path]=/var/tmp/cache"
 --help (-h)            Display this help message
 --quiet (-q)           Do not output any message
 --verbose (-v|vv|vvv)  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
 --version (-V)         Display this application version
 --ansi                 Force ANSI output
 --no-ansi              Disable ANSI output
 --no-interaction (-n)  Do not ask any interactive question
```

It seems that there is no way to chose the account locale when creating an user via cli. This is a big inconvenience because we only deploy the languages that we need, not en_US and then the users created via cli don't have access to the backend which is serving en_US for them 
### Preconditions

any magento2 installation
### Steps to reproduce

php bin/magento admin:user:create --admin-user mynampe --admin-password mypass123 --admin-email myemail@emaeil.com --admin-firstname me --admin-lastname surname

creates with default locale en_US and I cannot change it via cli.

If you go into backend on a production website that deploy only en_GB it will show no styles or js.
### Expected result

a flag to select locale like --locale=en_GB

**Comment by veloraven** [comment]
@ClaudiuCreanga thank you for your report.
Please, identify which version of Magento you are running.
I can see a bug and an improvement in your description:
- no styles or js - is a bug;
- adding of additional parameter - is improvement.

Please copy the proposed improvement to the [Magento 2 Feature Requests and Improvements forum](https://community.magento.com/t5/Magento-2-Feature-Requests-and/idb-p/feature-requests), you can find more information [here](https://community.magento.com/t5/News-Announcements/Improvements-to-GitHub-Management/m-p/44572#M96).
And here in GitHub we will have a look at the issue with js. But we need to know Magento version you use.

**Comment by ClaudiuCreanga** [comment]
Thanks for your reply, I will submit it there then. 

I'm using 2.1.1

If you are in production mode and you only deploy en_GB locale (make sure you first delete en_US in pub/static if you previously deployed en_US) and then you have an user with the admin locale en_US then when that user tries to log into backend he will have 404 for css and js and in page source the links are all for pub/static/.../en_US/ which doesn't exist.

I don't know if it is really a bug.

**Comment by BaDos** [comment]
Hi, @ClaudiuCreanga!

Thank you for your feedback.
Internal ticket `MAGETWO-60631` was created.

**Comment by magento-engcom-team** [comment]
@ClaudiuCreanga, thank you for your report.
The issue is already fixed in develop branch, 2.2.0
